source/docs/articles/sites/domains/gandi-pantheon-pointing-your-dns.md

```diff
@@ -6,41 +6,40 @@ category:
-
-<div class="alert alert-info" role="alert">
-<h4>Note</h4>
-This guide assumes you have already registered your domain through Gandi.net.</div>
+[Gandi](https://www.gandi.net/) provides you with a nice, clean list of your domains. Click on the domain that you are interested in pointing to Pantheon.
 
 ## Domain Configuration Page
 
-The domain portal contains all relevant registration information for your domain. For pointing DNS, we are concerned with the DNS configuration section on the right.
-
-In traditional webhosting, you could often times just point your domain name to your hosts nameservers and, after propagation, all would resolve as desired. Proper resolution for your Pantheon site depends on CNAMEs and A records. To configure these entries, you'll have to edit the DNS Zone Record. Click **Edit the Zone** to proceed to the configuration page.<br />
- ![Gandi DNS Zones](/source/docs/assets/images/desk_images/198816.png)​
-## Configuring Zone Records on Gandi
+The domain portal contains all relevant registration information for your domain. Proper resolution for your Pantheon site depends on CNAMEs and A records. To configure these, you'll have to edit the DNS Zone Record. Click **Edit the Zone** to proceed to the configuration page.
+
+## Configure Zone Records on Gandi
 
 Gandi requires you to make a copy of your default Zone Record to ensure that you don't botch your default configuration. Click **Make New Version** to create a new version to edit.
 
-Gandi has two edit modes: "Normal" and "Expert". For this tutorial, we'll be using "Expert" editing mode, which simply allows us to edit the Zone Record like a text file instead of inputting values into the Gandi fields and letting them do the formatting.
+Gandi has two edit modes: Normal and Expert. Select the **Expert** editing mode to edit the Zone Record as a text file, instead of inputting values into the Gandi fields and letting them do the formatting.
 
-We are concerned with two lines here. The A record, which reads something to the tune of:
+You'll need to edit the A record and CNAME. The A record shows something like:
 ```nohighlight
 @ 10800 IN A 217.70.184.38
 ```
-
And the www CNAME:

+
And the www CNAME shows:

 ```nohighlight
 www 10800 IN CNAME webredir.vip.gandi.net.
 ```
 
-We will be pointing our A record to 50.56.49.247, as per the [Pantheon DNS document](/docs/articles/going-live#get-dns-record-and-update-your-dns). You will need to change the A record line in the zone file to the following:
+1. Point the A record in the zone file to 50.56.49.247:
 ```nohighlight
 @ 300 IN A 50.56.49.247
 ```
-
Change the www CNAME to match the following:

-<div class="alert alert-warning" role="alert">
-<h4>Note</h4>
-Be sure to include the trailing "."</div>
+
2. Next, change the www CNAME to match the following:

 ```nohighlight
 www 300 IN CNAME edge.live.getpantheon.com.
 ```
+
+<div class="alert alert-warning" role="alert">
+<h4>Note</h4>
```

Comment:
@nataliejeremy 
No, it looks like Gandi doesn't allow adding a CNAME for the bare domain: 

> When not to use a CNAME
>You cannot create a CNAME record for a bare domain. It can only be on a subdomain.
from https://wiki.gandi.net/en/dns/zone/cname-record

Ping me in Slack or the office if you want to discuss.